README.md

```diff
@@ -1,6 +1,209 @@
+
+<p align="center">
+  <a href="https://github.com/rasmusmarak/TOSS">
+    <img src="docs/source/TOSS_Optimization_Scheme.pdf" alt="Scheme"  width="910" height="459">
+  </a>
+  <p align="center">
+    Trajectory Optimisation of a Spacecraft Swarm
+    <br />
+    <a href="https://github.com/rasmusmarak/TOSS/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rasmusmarak/TOSS/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+### Built With
+
+This project is based on:
+
+- [polyhedral-gravity-model](https://github.com/esa/polyhedral-gravity-model) a fast, parallelised version of the polyhedral gravity model implemented in C++ along with a python interface.
+- [DESolver](https://github.com/Microno95/desolver) a python library fro solving Initial Value Problems using numerical integrators. 
+- [PyGMo](https://esa.github.io/pygmo2/index.html) a scientific python library for massively parallel optimization managed by the Advanced Concepts Team at the European Space Agency
+
+For more details than provided by TOSS on these libraries, please refer to their docs.
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This is a brief guide how to set up TOSS.
+
+### Installation
+
+### conda
+The recommended way to install TOSS is via [conda](https://docs.conda.io/en/latest/) / [mamba](https://github.com/conda-forge/miniforge#mambaforge) using
+
+```
+conda install toss -c conda-forge
+
+```
+
+### Building from source
+
+Alternatively, to use the latest code from this repository make sure you have all the requirements installed and then clone the [GitHub](https://github.com/rasmusmarak/TOSS.git) repository as follows ([Git](https://git-scm.com/) required):.
+
+```
+git clone https://github.com/rasmusmarak/TOSS
+```
+
+To install TOSS you can use [conda](https://docs.conda.io/en/latest/) as follows:
+
+```
+cd TOSS
+conda env create -f environment.yml
+```
+
+This will create a new conda environment called `toss` and install the required software packages.
+To activate the new environment, you can use:
+
+```
+conda activate toss
+```
+
+
+### Test
+
+After cloning the repository, developers can check the functionality of TOSS by running the following command in the `TOSS/tests` directory:
+
+```sh
+pytest
+```
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+### Config
+
+TOSS uses a central config file which is passed through the entire program. The default config parameters can be seen [here](https://github.com/rasmusmarak/TOSS/blob/2501351468eb362fa9ec8217b2fc426db37fca9a/toss/resources/default_cfg.toml). 
+
+**[Coming Soon]** Practical usage of the config files will soon be added here through an explanatory notebook.
```

Comment:
Maybe create an issue for it if you plan to do it? :)

Reply:
There is one in #48 and I plan to do this as soon as possible, but likely not until right after the thesis has been officially finished.